schema.rst

```diff
@@ -62,6 +62,11 @@ Metadata Schema for the Persistent Identification of Scientific Measuring Instru
 +-------+----------------------------+------------+-----+------------------------+------------------------+
 | 5.2.1 | manufacturerIdentifierType | O          | 1   | Type of the identifier | Free text              |
 +-------+----------------------------+------------+-----+------------------------+------------------------+
+| 5.3   | modelName                  | R          | 0-1 | Name of the model or   | Free text              |
```

Comment:
Should we specify here for allowed values "Free text, should be identifiers or resource locators"?

Reply:
I suggest to avoid making it too complicated.  To link the the model's web page at the manufacturer, one can still use `RelatedIdentifier` with `relationType` `IsDescribedBy` or `References`.

Reply:
We can leave it at this for now, but I think model is too important and ubiquitously related to an instrument instance to be considered a related identifier. I think the importance of the relationship between instance and model deserves to be more explicit (a bit like the discussion on serial numbers). In other words, a computer agent should be able to easily follow the link from instance to model. The "modelName" attribute allows for that but I think we should encourage to use unambiguous names as much as possible, also for models.